scripts/data_OEP_registration.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pathlib
+
+import environ_setup
+import requests
+
+environ_setup.setup()
+
+MODEX_URL = "https://modex.rl-institut.de/create_table/"
+
+path = pathlib.Path(
```

Comment:
Path points to your personal path. I would recommend creating a function which gets pathname from user and goes on from there. (Same for other script files)

Reply:
I have replaced the "Path" variable with a descriptive string and included instructions within the documentation on how the user should input their desired folder path.

It would be possible to read the path from the Metadata resource information, while this would be possible, it would require additional changes to the code. If this is something that you would like me to implement, please let me know.